.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,118 @@
+name: Uwazi CI
+
+on: [push]
+
+jobs:
+  
+  uinit_test:
+    runs-on: ubuntu-latest
+    services:
+      elasticsearch:
```

Comment:
Do you need elastic for unit tests?

Reply:
Yes, for the backend test for the search